nomad-cluster-security.md

```diff
@@ -0,0 +1,14 @@
+# Nomad Cluster Security
+
+1. What is the noamd cluster size?
```

Comment:
Spell and grammar check the document, please. Use Google Docs for writing it?

Reply:
Spelling correction done.